tests/prepare_test_data.py

```diff
@@ -0,0 +1,132 @@
+import json
+import os
+import shutil
+import tarfile
+import tempfile
+
+import git
+
+from system.plugins.gosymbolextractor.extractor import GoSymbolExtractor
+
+
+DESTDIR = 'testdata'
+
+PROJECTS = [
+    {
```

Comment:
Actually, maybe it would be better to remove this and just generate exported API for all projects.